### DISCUSSION TITLE
Version number doesn't show anywhere

### DISCUSSION THREAD

**Discussion by Normandeau** [ExistentialAudio/BlackHole#309] [General]
Version 0.2.7 vs 0.2.8
As we are currently trying to debug a problem with audio glitches in the recording with SpatGRIS, we had to switch between these two recent versions of BlackHole. But apart from the installer, it is not possible to know which version is installed in the system. It would help to know that.

**Comment by devinroth** [comment]
In terminal type

"defaults read /Library/Audio/Plug-Ins/HAL/BlackHole.driver/Contents/Info.plist CFBundleShortVersionString”

Or

"defaults read /Library/Audio/Plug-Ins/HAL/BlackHole128ch.driver/Contents/Info.plist CFBundleShortVersionString"

That will show the version number. I changed the .driver names to include the channel in 0.2.8 so you’ll need to try both commands.

Devin
